rpc/src/lib.rs

```diff
@@ -42,20 +47,18 @@ where
 {
     // Grab trace information
     /////////////////////////
-    let traces = provider
-        .raw_request::<_, Vec<ZeroTrace>>(
+    let tx_results = provider
+        .raw_request::<_, Vec<ZeroTxResult>>(
             "debug_traceBlockByNumber".into(),
             (target_block_id, json!({"tracer": "zeroTracer"})),
         )
         .await?;
 
-    let (txn_info, mut pre_images) =
-        traces
-            .into_iter()
-            .partition_map::<Vec<_>, Vec<_>, _, _, _>(|it| match it {
-                ZeroTrace::Result(it) => Either::Left(it),
-                ZeroTrace::BlockWitness(it) => Either::Right(it),
-            });
+    // Grab block witness info (packed as combined trie pre-images)
+    ///////////////////////////////////////////////////////////////
```

Comment:
Not convinced by the lines of `\\\\\`. I know they already exist in the code base though

Reply:
Just followed the existing code. Happy to remove them.

Reply:
Yeah I had told Aatif on his PR that this wasn't the convention we had on `zero-bin` but it got merged nonetheless. I am not against it (not my preference though), but I wouldn't mind too much as long as we get consistent across the whole codebase.

Reply:
I'm happy for a vote on this to take the consensus on the `zero_dev` channel.